docs/tutorials/run-ollama-tutorial.md

```diff
@@ -0,0 +1,125 @@
+without the need to specify the specific machine ID.
+
+Here's what this command does:
+1. fair docker run: Pulls image and starts the container.
+2. --name ollama : Specifies the name for the container.
+3. -p 11434:11434: Maps port 11434 on your server to port 11434 on the container,
+allowing you to access the Ollama.
+4. ollama/ollama:latest : Specifies the docker image to be run.
+
+After running this command, you'll see output similar to the following:
+
+```bash
+# b3501dc206e22f10e69e3c597d7a912423aa84dc665c0fc749b9c5a40c32632b
+```
+
+## Ensure Ollama is up and running
+
+You can add the port number ahead of your GPU URL and you will see
+"Ollama is running" on your system. You can also make a curl request 
+to make sure that the model is up and running using the following 
+command :
+
+```bash
+curl <node-IP-address>/11434
+```
+
+Replace the node-IP-address with the IP address received while renting the
+Fair Compute GPU or you can also run the following command to get 
+the IP address of the machine.
+
+```shell
+fair cluster info
+```
+
+## Run inferences with LLMs using Ollama container
+
+To interact with LLM using the running fair docker container for 
+ollama , run the following command :
+
+```shell
+fair docker exec -it ollama ollama run llama3
+```
+
+For this tutorial purpose, we are running LLama-3 by Meta AI, which
+is one of the most powerful LLM in the open-source ML community.
+You can also run inferences on any LLM from your choice supported
+by the Ollama library. You can find the list of models supported by
+Ollama [model hub](https://ollama.com/library).
+
+After running the above command you will see output similar to the 
+following :
+
+```shell
+pulling manifest 
+pulling 2af3b81862c6... 100% ▕████████████████▏ 637 MB                         
+pulling af0ddbdaaa26... 100% ▕████████████████▏   70 B                         
+pulling c8472cd9daed... 100% ▕████████████████▏   31 B                         
+pulling fa956ab37b8c... 100% ▕████████████████▏   98 B                         
+pulling 6331358be52a... 100% ▕████████████████▏  483 B                         
+verifying sha256 digest 
+writing manifest 
+removing any unused layers 
+success 
+>>> Send a message (/? for help)
+```
+
+## Use API to make inferences with LLMs using OLlama
+
+You can also use API calls to pull and chat with the model.
+Here's how you do it :
+
+Pull the model using the following command:
+
+```shell
+curl http://localhost:11434/api/pull -d '{ 
+  "name": "llama3"
+}'
+```
+
+## Interact with model making API calls
+
+Run the following command specifying the name of 
+the model pulled as well as the prompt for the 
+model to generate response
+
+```shell
+curl http://localhost:11434/api/generate -d '{
+  "model": "llama3",
+  "prompt": "Explain large language models"
+}'  
+```
+
+That's it. Now you can use run Ollama locally and 
+avoid paying for expensive cloud GPU instances using
```

Comment:
In this tutorial we're renting the node, so not running ollama locally and are paying for cloud compute. Makes sense to remove this sentence or rephrase it.